api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -465,6 +465,159 @@ function rewardReducer(network, validators, reward, reducers) {
   return parsedRewards
 }
 
+function democracyProposalReducer(
+  network,
+  proposal,
+  totalIssuance,
+  blockHeight
+) {
+  return {
+    id: proposal.index,
+    networkId: network.id,
+    type: `text`,
+    title: `Democracy #${proposal.index}`,
+    description: proposal.description,
+    creationTime: undefined,
```

Comment:
not possible?

Reply:
Maybe get the height and query the block for the time?

Reply:
I need to check

Reply:
OK, this is basically what I have to do and after this I would call this ready